Azure/sonic-buildimage
# [Inventec] Add support for D6332 platform

**- Why I did it**
Add new platform ( Inventec D6332) on master branch

**- How I did it**
Support D6332 platform, and upgrade kernel to 4.19

**- How to verify it**
- Basic traffic tests
- Run "show interface status" "show interface transceiver presence" to check transceiver/port status
- Run "show platform fan", "show temperature", "show environment" ...etc to check device status
Please refer to the test report for more details
[d6332_master.pdf](https://github.com/Azure/sonic-buildimage/files/5165849/d6332_master.pdf)

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

Support platform Inventec D6332

**- A picture of a cute animal (not mandatory but encouraged)**
